src/status_im/contexts/profile/settings/view.cljs

```diff
@@ -29,12 +30,13 @@
 
 (defn- footer
   [logout-press]
-  [rn/view {:style style/footer-container}
-   [quo/logout-button {:on-press logout-press}]])
+  [rf/delay-render
+   [rn/view {:style style/footer-container}
+    [quo/logout-button {:on-press logout-press}]]])
 
 (defn- get-item-layout
   [_ index]
-  #js {:length 48 :offset (* 48 index) :index index})
+  #js {:length 100 :offset (* 100 index) :index index})
```

Comment:
I'm wondering why this has been changed. Were the dimensions wrong? does this affect the UI?

Reply:
it's just a random number, i just wanted to reserve more space

Reply:
I believe `getItemLayout` should reflect the actual item dimensions:

> getItemLayout is an optional optimization that allows skipping the measurement of dynamic content if you know the size (height or width) of items ahead of time. 

https://reactnative.dev/docs/flatlist#getitemlayout

So maybe changing this for random values might have an unexpected behavior

Reply:
yes it should but here its used only for first "empty" rendering, and then layout will be calculated by real item size, in this case it will be different for items

Reply:
Sorry, I'm not understanding how this will be done.

Do you mean that if we pass wrong dimensions, RN will only use them for the first rendering and in following renders they will be automatically calculated? 
Does RN know that calculations are wrong?

Many questions, just not sure what the approach used here will do.